packages/amplify-codegen/package.json

```diff
@@ -25,7 +25,7 @@
     "@aws-amplify/graphql-docs-generator": "4.2.1",
     "@aws-amplify/graphql-generator": "0.4.6",
     "@aws-amplify/graphql-types-generator": "3.6.0",
-    "@graphql-codegen/core": "2.6.6",
```

Comment:
Without this change, our code depends on 2 versions of core. Applying this so that we only have a single version in our dependency list for the monorepo